.vscode/extensions.json

```diff
@@ -0,0 +1,9 @@
+{
+  // See http://go.microsoft.com/fwlink/?LinkId=827846
+  // for the documentation about the extensions.json format
+  "recommendations": [
+    // Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
+    "editorconfig.editorconfig",
```

Comment:
Personally, I don't use `editorconfig` and use `goreturns` for formating go files.

Reply:
Maybe something we should do, as it also eliminates the need for the fix-line-endings make targets.